.bump-version.js

```diff
@@ -0,0 +1,56 @@
+const replace = require("replace");
+const yaml = require('js-yaml');
+const fs = require('fs');
+const {execSync} = require('child_process')
+
+const versionRegex = /\d+\.\d+\.\d+(-\w+)?/i;
+const newVersion = process.env.SEM_VER;
+
+function checkVersion(semver){
+
+    if(semver == undefined || semver == '') {
+        console.log("semver cannot be blank");
+        process.exit(1);
+    }
+
+    if(semver.match(versionRegex) == undefined) {
+        console.log(`semver: ${semver} must be Semantic Version <Major>.<Minnor>.<Patch>[-<type>], examples: '0.20.0', '0.20.0-rc1'`)
+        process.exit(1);
+    }
+
+    return semver;
+}
+
+// check is a valid version
+checkVersion(newVersion);
+
+console.log(`Bumping version to: ${newVersion}`);
+
+// bumping version using replace for 'packages/relay/package.json', 'packages/server/package.json', 'docs/openrpc.json'
+replace({
+    regex: '\"version\": \"\\d+\\.\\d+\\.\\d+(-\\w+)?\"',
+    replacement: `"version": "${newVersion}"`,
+    paths: ['packages/relay/package.json', 'packages/server/package.json', 'docs/openrpc.json'],
+    recursive: false,
+    silent: false,
+});
+
+// bump docker-compose.js version
```

Comment:
No, the intention is to support all 3 types of version bumps (GA, RC and SNAPSHOTS), talking with Luke and taking a look at old bump PRs determined that the only one that change are the SNAPSHOTS since they don't change "main" in files `README.md` and `docker-compose.yml`

So I have made the change to add another parameter to the script to determine if is a snapshot bump, and bump those 2 files only when is not a snapshot.